packages/aws-cdk-lib/aws-codebuild/lib/linux-arm-lambda-build-image.ts

```diff
@@ -21,10 +21,16 @@ interface LinuxArmLambdaBuildImageProps {
 export class LinuxArmLambdaBuildImage implements IBuildImage {
   /** The `aws/codebuild/amazonlinux-aarch64-lambda-standard:nodejs18` build image. */
   public static readonly AMAZON_LINUX_2_NODE_18 = LinuxArmLambdaBuildImage.fromCodeBuildImageId('aws/codebuild/amazonlinux-aarch64-lambda-standard:nodejs18');
+  /** The `aws/codebuild/amazonlinux-aarch64-lambda-standard:nodejs20` build image. */
+  public static readonly AMAZON_LINUX_2023_NODE_20 = LinuxArmLambdaBuildImage.fromCodeBuildImageId('aws/codebuild/amazonlinux-aarch64-lambda-standard:nodejs20');
```

Comment:
Why did you decide to name these `AMAZON_LINUX_2023` when the others are named `AMAZON_LINUX_2`? It looks like the codebuild image id is largely the same `amazonlinux-aarch64-lambda-standard` so I'm curious why this decision was made.

Reply:
It seemed sensible to use that as per the `platform` indicated in this [listing](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html#lambda-compute-images). 

Another, although breaking change, would be to rename all with `AMAZON_LINUX` prefix as it would be more in line with the image id as you pointed out.

Reply:
@kaizencc Do you have any suggestions on how to proceed with this?

Reply:
I agree that it doesn't make any sense to use the `AMAZON_LINUX_2` prefix as the new image is not based on Amazon Linux 2. If the `AMAZON_LINUX_2023` prefix is unacceptable, perhaps just using `AMAZON_LINUX` for the new images would be the best choice. Personally I wouldn't rename the old images to avoid breaking existing setups.

Reply:
Nope. Makes sense now. Just based on the image id it was hard to tell though